README.md

```diff
@@ -28,6 +28,7 @@ Current maintainers: @cosmo0920
   + [time_precision](#time_precision)
   + [time_key](#time_key)
   + [time_key_exclude_timestamp](#time_key_exclude_timestamp)
+  + [include_timestampt](#time_key_exclude_timestamp)
```

Comment:
nit: Typo?

```log
include_timestampt ->
include_timestamp
                 -
```

Reply:
I got a new keyboard and am constantly adding a t onto the ends of things, thanks for the catch :P